js/components/wysiwyg.js

```
@@ -51,6 +51,49 @@ Fliplet.FormBuilder.field('wysiwyg', {
+        .toLowerCase();
+    },
+
+    /**
+     * Gets a nested property from an object using dot notation
+     * @param {Object} obj - object to get property from
+     * @param {String} path - dot-separated path
+     * @returns {*} property value
+     */
+    getNestedProperty: function(obj, path) {
+      return path.split('.').reduce(function(current, key) {
+        return current && current[key];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)